cypress/support/commands.js

```diff
@@ -23,3 +23,12 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+    cy.get(`[placeholder="${placeholder}"]`);
+  });
+  
+  Cypress.Commands.add('assertPageUrl', (url) => {
+    cy.url()
+        .should('include', Cypress.config().baseUrl + url)
+  });
\ No newline at end of file
```

Comment:
Please add a newline at the end of the file to follow best practices and improve code readability 😊.